src/libstd/process.rs

```diff
@@ -149,12 +152,16 @@ impl fmt::Debug for Child {
     }
 }
 
-/// A handle to a child process's stdin.
+/// A handle to a child process's standard input (stdin).
 ///
 /// This struct is used in the [`stdin`] field on [`Child`].
 ///
+/// When an instance of `ChildStdin` is [dropped], the `ChildStdin`'s underlying
+/// operating system file descriptor will be closed.
```

Comment:
'underlying operating system file descriptor' is a mouthful. very open to other terminology here.

Reply:
"file handle", maybe? that's just trading one OS's terminology for another's

Reply:
i'm 🆒 with 'file handle'. do you think "underlying operating system file handle" or "underlying file handle" or "file handle" or something else?

Reply:
"underlying file handle" was what i had in mind

Reply:
thanks for the idea! 'underlying file handle' incorporated in the latest force push